### ISSUE TITLE
Error spamming

### ISSUE THREAD

**Issue by CrafterBotOfficial** [KyleTheScientist/Bark#1]
When the player connects to a lobby the Bark spams this error around 10 times.

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Bark.Modules.BarkModule.OnEnable () (at <387329cbe5234c7a8986399630213367>:0)
Bark.Modules.Boxing.OnEnable () (at <387329cbe5234c7a8986399630213367>:0)
UnityEngine.GameObject:AddComponent()
Bark.GUI.MenuController:Awake()
UnityEngine.GameObject:AddComponent()
Bark.Plugin:Setup()
Bark.Plugin:RoomJoined(String)
System.Linq.Expressions.Interpreter.LightLambda:RunVoid1(String)
Utilla.GamemodeManager:OnRoomJoin(Object, RoomJoinedArgs)
Utilla.EventHandlerExtensions:SafeInvoke(EventHandler`1, Object, RoomJoinedArgs)
Utilla.Events:TriggerRoomJoin(RoomJoinedArgs)
Utilla.UtillaNetworkController:OnJoinedRoom()
Photon.Realtime.MatchMakingCallbacksContainer:OnJoinedRoom()
Photon.Realtime.LoadBalancingClient:OnEvent(EventData)
ExitGames.Client.Photon.PeerBase:DeserializeMessageAndCallback(StreamBuffer)
ExitGames.Client.Photon.EnetPeer:DispatchIncomingCommands()
ExitGames.Client.Photon.PhotonPeer:DispatchIncomingCommands()
Photon.Pun.PhotonHandler:Dispatch()
Photon.Pun.PhotonHandler:FixedUpdate()
```

Then it spams: ( around 10 times )
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Bark.Modules.BarkModule.OnDisable () (at <387329cbe5234c7a8986399630213367>:0)
Bark.Modules.XRay.OnDisable () (at <387329cbe5234c7a8986399630213367>:0)
UnityEngine.Behaviour:set_enabled(Boolean)
Bark.Modules.BarkModule:Start()

```

These errors do not appear to be doing anything negative interms of gameplay. However they are very annoying when going through the console.

**Comment by KyleTheScientist** [comment]
I will look into this soon. Probably just an issue with trying to destroy some of the components twice.

**Comment by KyleTheScientist** [comment]
I believe this should be resolved as of 1.1.0.
